packages/core/src/external/carequality/error.ts

```diff
@@ -78,6 +78,7 @@ function constructBaseErrorResponse(
     patientId: payload.patientId,
     timestamp: payload.timestamp,
     responseTimestamp: new Date().toISOString(),
+    signatureConfirmation: payload.signatureConfirmation,
```

Comment:
Is this fine to send to Sentry and/or log (re: sensitive information)?

Reply:
Its not sensitive